<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>53</y>
          <width>771</width>
          <height>331</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QVBoxLayout" name="main_additionalOptionsVLayout">
           <item>
            <layout class="QVBoxLayout" name="main_presetVLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="main_presetTitleLabel">
               <property name="styleSheet">
                <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
               </property>
               <property name="text">
                <string>Preset</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="main_presetHLayout">
               <item>
                <layout class="QFormLayout" name="main_presetFormLayout">
                 <item row="0" column="1">
                  <widget class="QComboBox" name="main_presetComboBox"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="main_presetChooseLabel">
                   <property name="text">
                    <string>Preset</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTextEdit" name="main_presetDescriptionTextEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>800</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="main_additionalTitleLabel">
             <property name="styleSheet">
              <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>Additional options</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="main_additionnalOptionsCheckBoxesHLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="main_showDiscordCheckBox">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Show on Discord</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QCheckBox" name="main_runStartupCheckBox">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Run on save &amp; exit</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="config">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="presets">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="presets_listvLayout">
           <item>
            <widget class="QLineEdit" name="presets_searchLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 1px solid black;
color: rgb(109, 109, 109);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="presets_listView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>500</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="presets_buttonshLayout">
             <item>
              <widget class="QPushButton" name="presets_newButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: rgb(0,49,156);
	color: white;
	border: no-border;
	height: 25px;

}
QPushButton:pressed { 
	background-color: rgb(74,75,180);
}</string>
               </property>
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="presets_modifyButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: rgb(0,49,156);
	color: white;
	border: no-border;
	height: 25px;

}
QPushButton:pressed { 
	background-color: rgb(74,75,180);
}</string>
               </property>
               <property name="text">
                <string>Modify</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="presets_deleteButton">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: rgb(0,49,156);
	color: white;
	border: no-border;
	height: 25px;

}
QPushButton:pressed { 
	background-color: rgb(74,75,180);
}</string>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="presets_infosvLayout">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="presets_infos_presetLabel">
               <property name="text">
                <string>Preset </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="presets_infos_presetName">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="presets_infos_presetCategoryLabel">
               <property name="text">
                <string>Category </string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="presets_infos_presetCategoryName">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="presets_infos_presetDomTypeLabel">
               <property name="text">
                <string>Dom type</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="presets_infos_presetDomType">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="presets_infos_presetDomNameLabel">
               <property name="text">
                <string>Dom Name</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="presets_infos_presetDomName">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="presets_spacerPresetCategory">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="presets_StartupRunLabel">
               <property name="text">
                <string>Run on startup</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="presets_infosPopupsLabel">
               <property name="text">
                <string>Popups</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="presets_StartupRun">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="presets_infosVideosLabel">
               <property name="text">
                <string>Videos</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="presets_infosAudioLabel">
               <property name="text">
                <string>Audios</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="presets_infosWebsitesLabel">
               <property name="text">
                <string>Websites</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="presets_infosPopups">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="presets_infosVideos">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QLabel" name="presets_infosAudios">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLabel" name="presets_infosWebsites">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="presets_infos_presetDomName_2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>190</y>
          <width>234</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="about">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>50</y>
          <width>118</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="about_vLayout">
         <item>
          <widget class="QLabel" name="about_appNameLabel">
           <property name="text">
            <string>SubbyWare</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="about_versionLayout">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="about_version_label">
             <property name="text">
              <string>Version :</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="about_vnumber_label">
             <property name="text">
              <string>0.1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="about_author_layout">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="about_author_label">
             <property name="text">
              <string>Author :</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="aboutauthorName_label">
             <property name="text">
              <string>Mimifan</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
